### DISCUSSION TITLE
Finetuning model via cloud

### DISCUSSION THREAD

**Discussion by Lolkekxd** [erew123/alltalk_tts#278] [Q&A]
Is it possible to train and finetune models via cloud GPU services like runpod? I'm not super tech savvy so I apologize if this is known, but looking around I have not found how to connect them together.

**Comment by erew123** [comment]
Hi @Lolkekxd 

The answer is a yes and a no. There is no reason you cannot run finetuning on an cloud based service, however I have not had time to fully setup and work through all the possible issues/problems of doing so.

The one service that is nearest to be able to do that is Google Colab. You would open the `[googlecolab.ipynb](https://github.com/erew123/alltalk_tts/blob/alltalkbeta/googlecolab.ipynb) in Google colab and there is an option to run the finetuning there. I have not had time to test that out though yet, however, I do know that someone has used Google Colab for Finetuning, but there may be technical challenges they overcame/setup or settings needed.

Thanks
